docs/UserGuide.md

```diff
@@ -335,6 +336,7 @@ Opens a window at the side with the full details of the specified person from th
 
 * `view n/JohnDoe` opens a window at the side showing the full details of the person named `JohnDoe`.
 * Calling `view n/JohnDoe` followed by `view` closes the window showing the full contact details of `JohnDoe`.
+  ![viewing John Doe's contact detail in the view window](images/viewWindowScreenshot.png)
```

Comment:
line 338, 337 the JohnDoe need spacing i think (as shown in picture)

Reply:
Good catch! will update it!